text/0005-architecture-reviews.md

```diff
@@ -0,0 +1,135 @@
+
+2. The team produces a **Review Packet** designed to answer the question identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key questions and the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the answer are likely to be higher.
+The Review Packet includes:
+  * A lay summary of the problem space and the stakeholders which focused on defining a shared language and identifying the key forces behind the problem
+  * A brief explaining what the team proposes which should read more like an encyclopedia entry than a marketing document.  The brief should identify:
+    - the forces on the system
+    - the proposed system’s essential architecture
+    - scenarios that exercise the relevant forces against the architecture
+    - how the proposed system handles these scenarios
+  * A competitive analysis suggesting alternatives, costs, and opportunities.
+
+3. The chair creates a list of questions to be discussed during the review. These questions should be:
+  * Open-ended
+  * Made available to the the team prior to the review so they can prepare answers
+
+4. The chair convenes a review meeting. The discussions of this meeting should be carefully minuted. The meeting comprises:
+  * The team making the proposal
+  * The chair
+  * (For Design Reviews) other senior engineers who will provide valuable input.
+
+The review meeting should be a discussion of the issues, but should avoid feeling pressured to make a decision, the goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review. The meeting may decide to alter the questions and undergo a subsequent review. For questions that are go/no-go in nature the decision does not need to be taken during the meeting, although we should be very open about conversations that happen after the review meeting.
+
+The ideal review process scales well. The same basic system should work for a quick 2 person decision over the best way to design a certain feature, and for a critical organization wide decision about a path to take. The term “team” is used above although we strongly recommend design reviews for smaller questions. If you feel yourself wondering if some design is best, it should be easy to valuable to perform a review.
+
+## Rationale
+
+### Key considerations
+
+* What are the desired **outputs** of the review process?
+* Which **audience** is intended to consume the outputs of the review process?
+* What committee **structure** is best positioned to deliver those outputs?
+* What are the **inputs** that the committee should evaluate?
+* What size and **maturity** should projects achieve before undergoing review?
+* When should review results be **disseminated**?
```

Comment:
And:

* What process will most encourage participants to view review as an asset and as an opportunity to seek out, not as an obstacle to avoid?
* What kinds of conduct are necessary in order for the process to be inclusive and constructive, rather than being optimized for certain privileged groups with particular communication styles (_e.g._, verbally argumentative white male senior engineers)?